demo.py

```diff
@@ -23,24 +23,29 @@ def main():
   # Define the supply chain
   #############################################################################
   """
+  # Create keys
+  print "Generate keypair for Alice..."
```

Comment:
I created an issue to address python2/3 compatibility https://github.com/in-toto/in-toto/issues/23.